tests/commands/test_session.py

```diff
@@ -65,6 +66,20 @@ def _set_mocks_for_json_file_exists(mock_path, does_it_exist=True):
     return mock_path
 
 
+def _set_mocks_for_creds_file(mock_file):
+    # patch("builtins.open", mock_open(read_data="dummypassword"))
```

Comment:
are these going to be added back, or are they no longer needed?

Reply:
yeah we should lose all the commented lines here